src/Application/Common/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,11 @@
+namespace MRA.AssetsManagement.Application.Common.Exceptions;
+
+public class NotFoundException : Exception
+{
+    public NotFoundException() { }
+    
+    public NotFoundException(string message)
```

Comment:
public NotFoundException(T entity, string id) : base("${typeof(T).Name] with provided ${id} was not found")